components/Toolbar/BlueprintsToolbar.js

```diff
@@ -1,79 +1,77 @@
+import { SortAlphaDownIcon, SortAlphaDownAltIcon } from "@patternfly/react-icons";
+
 import CreateBlueprint from "../Modal/CreateBlueprint";
 import ManageSources from "../Modal/ManageSources";
 
-const BlueprintsToolbar = (props) => (
-  <ToolbarLayout>
-    <div className="form-group">
-      {(props.sortKey === "name" && props.sortValue === "DESC" && (
-        <button
-          className="btn btn-link"
-          type="button"
-          disabled={props.emptyState}
-          onClick={() => props.sortSetValue("ASC")}
-        >
-          <span className="fa fa-sort-alpha-asc" />
-        </button>
-      )) ||
-        (props.sortKey === "name" && props.sortValue === "ASC" && (
-          <button
-            className="btn btn-link"
-            type="button"
-            disabled={props.emptyState}
-            onClick={() => props.sortSetValue("DESC")}
-          >
-            <span className="fa fa-sort-alpha-desc" />
-          </button>
-        ))}
-    </div>
-    <div className="toolbar-pf-action-right">
-      <div className="form-group">
-        <CreateBlueprint blueprintNames={props.blueprintNames} disabled={props.errorState} />
-        <div className="dropdown btn-group dropdown-kebab-pf">
-          <button
-            className="btn btn-link dropdown-toggle"
-            type="button"
-            id="dropdownKebab"
-            data-toggle="dropdown"
-            aria-haspopup="true"
-            aria-expanded="false"
-          >
-            <span className="fa fa-ellipsis-v" />
-          </button>
-          <ul className="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebab">
-            <li className={props.errorState ? "disabled" : ""}>
-              <ManageSources manageSources={props.manageSources} disabled={props.errorState} />
-            </li>
-          </ul>
-        </div>
-      </div>
-    </div>
-  </ToolbarLayout>
-);
+const BlueprintsToolbar = (props) => {
+  const [inputValue, setInputValue] = useState("");
+
+  const onInputChange = (newValue) => {
+    setInputValue(newValue);
+
+    const filter = {
+      key: "name",
+      value: newValue,
+    };
+    props.filterAddValue(filter);
+  };
+
+  const toolbarItems = (
+    <>
+      <ToolbarGroup alignment={{ default: "alignLeft" }}>
+        <ToolbarItem variant="search-filter">
+          <SearchInput
+            aria-label="blueprints search input"
+            onChange={onInputChange}
+            value={inputValue}
+            onClear={() => onInputChange("")}
+          />
+        </ToolbarItem>
+        <ToolbarItem variant="icon-button-group">
+          {(props.sortValue === "DESC" && (
+            <Button variant="plain" aria-label="sort asc" onClick={() => props.sortSetValue("ASC")}>
+              <SortAlphaDownIcon />
+            </Button>
+          )) ||
+            (props.sortValue === "ASC" && (
+              <Button variant="plain" aria-label="sort desc" onClick={() => props.sortSetValue("DESC")}>
```

Comment:
Same as above, `sort descending` might be better for screen readers...

Reply:
Good point!